### ISSUE TITLE
[fluentd-elasticsearch] PR #82 breaks metrics when using awsSigningSidecar

### ISSUE THREAD

**Issue by chrisob** [kiwigrid/helm-charts#119]
**Is this a request for help?**:
no
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
```console
$ helm version
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:11:31Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.3", GitCommit:"5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0", GitTreeState:"clean", BuildDate:"2019-06-06T01:36:19Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart in which version**:
fluentd-elasticsearch-3.0.1

**What happened**:
PR #82 introduced a bug ([among](https://github.com/kiwigrid/helm-charts/pull/103) [others](https://github.com/kiwigrid/helm-charts/pull/88)...) which [assigns service ports (including the prom metrics port) to the signing sidecar container](https://github.com/kiwigrid/helm-charts/blob/a2de7f9a3f98042385982087103e6b8ee647413d/charts/fluentd-elasticsearch/templates/daemonset.yaml#L174-L180), instead of the fluentd container.

In other words, when `awsSigningSidecar.enabled=true` and some service ports are enabled, the service ports incorrectly forward to the sidecar instead of fluentd-elasticsearch.

**What you expected to happen**:
Service ports should correctly route to the fluentd-elasticsearch container when the signing proxy sidecar is enabled. For example, when the `monitor-agent` service on port 24231 is enabled, fluentd-elasticsearch container metrics should be scrape-able from the service.

**How to reproduce it** (as minimally and precisely as possible):
```yaml
awsSigningSidecar:
  enabled: true

service:
  type: ClusterIP
  ports:
    - name: "monitor-agent"
      port: 24231
```

**Anything else we need to know**:
